src/components/hooks/use-livechat.js

```diff
@@ -0,0 +1,145 @@
+            clearInterval(cookie_interval)
+            clearTimeout(script_timeout)
+        }
+    }, [])
+
+    React.useEffect(() => {
+        if (isBrowser()) {
+            let customerSDK = null
+            const domain = window.location.hostname.includes('deriv.com')
+                ? 'deriv.com'
+                : 'binary.sx'
+            if (CustomerSdk.current) {
+                customerSDK = CustomerSdk.current.init({
+                    licenseId: livechat_license_id,
+                    clientId: livechat_client_id,
+                })
+            }
+            if (is_livechat_interactive) {
+                window.LiveChatWidget.on('ready', () => {
+                    let session_variables = {
+                        loginid: '',
+                        landing_company_shortcode: '',
+                        currency: '',
+                        residence: '',
+                        email: '',
+                    }
+                    if (is_logged_in) {
+                        const client_information = Cookies.get('client_information', {
+                            domain,
+                        })
+                        const {
+                            loginid,
+                            email,
+                            landing_company_shortcode,
+                            currency,
+                            residence,
+                            first_name,
+                            last_name,
+                        } = JSON.parse(client_information)
+
+                        session_variables = {
+                            ...(loginid && { loginid }),
+                            ...(landing_company_shortcode && {
+                                landing_company_shortcode,
+                            }),
+                            ...(currency && { currency }),
+                            ...(residence && { residence }),
+                            ...(email && { email }),
+                        }
+
+                        window.LiveChatWidget.call('set_session_variables', session_variables)
+                        if (email) {
+                            window.LiveChatWidget.call('set_customer_email', email)
+                        }
+                        if (first_name && last_name) {
+                            window.LiveChatWidget.call(
+                                'set_customer_name',
+                                `${first_name} ${last_name}`,
+                            )
+                        }
+                    } else {
+                        if (window.LiveChatWidget.get('chat_data')) {
+                            const chatID = window.LiveChatWidget.get('chat_data').chatId
+                            customerSDK?.deactivateChat({ chatId: chatID })
+                        }
+                        window.LiveChatWidget.call('set_customer_email', ' ')
+                        window.LiveChatWidget.call('set_customer_name', ' ')
```

Comment:
```suggestion
                        window.LiveChatWidget.call('set_customer_email', '')
                        window.LiveChatWidget.call('set_customer_name', '')
```

Should these be an empty string

Reply:
empty string will throw an error from livechat sdk, we need to pass space for emptying email and name

Reply:
I see, thank you for the clarification 🙏🏻